SMLHelper/Patchers/SpritePatcher.cs

```diff
@@ -10,7 +16,44 @@
 
     internal class SpritePatcher
     {
-        internal static void Patch()
+
+        internal static void PatchCheck(SpriteManager.Group group, string name)
+        {
+            if (string.IsNullOrEmpty(name) || !ModSprite.ModSprites.ContainsKey(group) || !ModSprite.ModSprites[group].TryGetValue(name, out _))
+                return;
+#if BELOWZERO
+            if (!SpriteManager.hasInitialized)
+                return;
+#endif
+            Dictionary<string, Sprite> atlas = GetSpriteAtlas(group);
+            
+            if (atlas != null && !atlas.TryGetValue(name, out _))
+                PatchSprites();
+        }
+
+        internal static void Patch(Harmony harmony)
+        {
+#if SN1
```

Comment:
Doesn't look like `SN1` is a valid build config
![image](https://user-images.githubusercontent.com/39146191/101281989-642e8f00-37a0-11eb-92bf-78fbba2f7d0f.png)

Reply:
24b985b